src/modules/bluetooth/a2dp/a2dp_sbc.c

```diff
@@ -251,7 +329,7 @@ pa_sbc_config_transport(pa_sample_spec default_sample_spec, const void *configur
 
     sample_spec->format = PA_SAMPLE_S16LE;
 
-    switch (config->frequency) {
+    switch (sbc_info->forced_frequency > 0 ? sbc_info->forced_frequency : config->frequency) {
         case SBC_SAMPLING_FREQ_16000:
```

Comment:
The `SBC_SAMPLING_FREQ_16000` is just a macro, the value of it would not be constant(in developing), should do a mapping for user inputed `forced_frequency`; Same as `forced_channel_mode`, `forced_allocation_method`, `forced_subbands`, `forced_block_length` at below.

Reply:
Sure. It just was easier for me to implement in a such way, because I don’t have much time right now. So, if there are no conceptual flaws in storing user-config values at sbc_info struct and the whole idea to control SBC codec parameters is OK - I'll do all required improvements after the new year holidays. In a couple of weeks I guess.

Reply:
I think it's OK.